ui/tests/acceptance/transit-test.js

```diff
@@ -6,101 +6,121 @@
 import { click, fillIn, find, currentURL, settled, visit, waitUntil, findAll } from '@ember/test-helpers';
 import { module, test } from 'qunit';
 import { setupApplicationTest } from 'ember-qunit';
-import { create } from 'ember-cli-page-object';
 import { v4 as uuidv4 } from 'uuid';
 
 import { encodeString } from 'vault/utils/b64';
 import authPage from 'vault/tests/pages/auth';
-import secretListPage from 'vault/tests/pages/secrets/backend/list';
-import consoleClass from 'vault/tests/pages/components/console/ui-panel';
-import { deleteEngineCmd, mountEngineCmd } from '../helpers/commands';
+import { deleteEngineCmd, mountEngineCmd, runCmd } from 'vault/tests/helpers/commands';
 
-const consoleComponent = create(consoleClass);
-const keyTypes = [
```

Comment:
I separated `keyTypes` into smaller groups with more similar attributes, removing the need for conditionals within the test. Hopefully this resolves some of the flakiness 🤞